0283_MoveZeroes.py

```diff
@@ -0,0 +1,19 @@
+class Solution(object):
+    def moveZeroes(self, nums):
+        nonzeroi=len(nums)-1
+        i=0
+        while i<nonzeroi:
+            if nums[i]!=0:
+                i+=1
+            else:
+                for j in range(i,nonzeroi):
+                    nums[j]=nums[j+1]
+                nums[nonzeroi]=0
+                nonzeroi-=1
+
+sol=Solution()
```

Comment:
In general, it's easier for you to have test function so that you don't need to manually enter values each time. This is especially important as you want to test with few different inputs, cover bigger spectrum of possibilities. Take a look at the test function from 0344_ReverseString.py.